core/scripts/local/DeployEMP.js

```diff
@@ -59,7 +59,7 @@ const deployEMP = async callback => {
 
     // Create a new EMP
     const constructorParams = {
-      expirationTimestamp: (await expiringMultiPartyCreator.VALID_EXPIRATION_TIMESTAMPS(0)).toString(),
+      expirationTimestamp: (await expiringMultiPartyCreator.VALID_EXPIRATION_TIMESTAMPS(6)).toString(),
```

Comment:
@ptare Since the current timestamp is almost at this chosen timestamp, it took me a while to debug some errors where I was trying to do contract actions post-expiry. I changed the hard-coded expiration timestamp in the convenience script here to give us more "breathing room"

But does highlight again to me the need for the user to know when the contract has expired